### PR TITLE
[PHPStan] Fix errors

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

File: models/DataObject/ClassDefinition/Data/Block.php [hunk 1]
```diff
@@ -958,7 +958,7 @@ public function save($object, $params = [])
      * @param DataObject\Concrete|DataObject\Localizedfield|DataObject\Objectbrick\Data\AbstractData|DataObject\Fieldcollection\Data\AbstractData $container
      * @param array $params
      *
-     * @return null
+     * @return mixed
```
Comment (weisswurstkanone) [inline models/DataObject/ClassDefinition/Data/Block.php]: array|null I guess

File: models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php [hunk 2]
```diff
@@ -131,7 +131,7 @@ public function save($object, $params = [])
      * @param DataObject\Concrete|DataObject\Localizedfield|DataObject\Objectbrick\Data\AbstractData|DataObject\Fieldcollection\Data\AbstractData $object
      * @param array $params
      *
-     * @return null
+     * @return array
```
Comment (weisswurstkanone) [inline models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php]: array|null

**Comment by blankse** [comment]
@weisswurstkanone I changed it

**Comment by weisswurstkanone** [comment]
thx!

**Pull request merged**